app/containers/NewApp/index.js

```diff
@@ -7,13 +7,15 @@
  */
 
 import React from 'react';
-import { Helmet } from 'react-helmet';
+import { Helmet } from 'react-helmet'; // QA: dlaczego niektóre importy są w nawiasach a inne nie?
```

Comment:
I've refered to the question in one of the previous answers. It's named import. Additionally to the previously send url, you can read all MDN sections regarding exports and imports.